amplify/backend/backend-config.json

```diff
@@ -7,17 +7,23 @@
     "bioMoleculesDB": {
       "service": "DynamoDB",
       "providerPlugin": "awscloudformation"
+    },
+    "psvfbaaaa36": {
```

Comment:
Are we still using this bucket? getting rid of this config from my PR fixed the build for me. 

But I see you have amplify/backend/storage/psvfbaaaa36/parameters.json and updated in amplify/team-provider-info.json as well - so if you need it you will need to look into how to fix it.
In my case, only this change got pulled in from amplify and I wasn't using it so it did not include it in my PR.

Reply:
Bless and thank you!